orchagent/swssnet.h

```diff
@@ -76,6 +77,53 @@ inline static sai_ip_prefix_t& copy(sai_ip_prefix_t& dst, const IpAddress& src)
     return dst;
 }
 
+static int getPrefixLenFromAddr(const uint8_t *addr, int len)
+{
+    int i = 0;
+    uint8_t non_zero = 0xFF;
+    for (i = len - 1; i >=0; i--)
+    {
+        if (addr[i] != 0)
+        {
+            non_zero = addr[i];
+            break;
+        }
+    }
+
+    if (non_zero == 0xFF)
+    {
+        return (i + 1) * 8;
+    }
+    else
+    {
+        int j = 2;
+        while(((non_zero >> j) & 0x1) == 0)
+        {
+            ++j;
+        }
+        return (i + 1) * 8 - (j + 1);
+    }
+    
+}
+
+inline static IpPrefix copy(const sai_ip_prefix_t& src)
+{
+    ip_addr_t ip;
+    switch(src.addr_family)
+    {
+        case SAI_IP_ADDR_FAMILY_IPV4:
+            ip.family = AF_INET;
+            ip.ip_addr.ipv4_addr = src.addr.ip4;
+            return IpPrefix(ip, getPrefixLenFromAddr(reinterpret_cast<const uint8_t*>(&src.mask.ip4), 4));
```

Comment:
Why do we need to calculate the getPrefixLenFromAddr? Can we just copy? Can we also make inline with other copy APIs? by passing IpPrefix as argument? 
https://github.com/Azure/sonic-swss/blob/e40c7367232eaea2374ea94fb99771326057c2a0/orchagent/swssnet.h#L44

Reply:
Will rename copy to convert